CONTRIBUTING.md

```diff
@@ -8,9 +8,11 @@ The Vite repo is a monorepo using pnpm workspaces. The package manager used to i
 
 To develop and test the core `vite` package:
 
+1. Run `pnpm i` in Vite's root folder
+
 1. Go to `packages/vite` and run `pnpm run dev`. This starts `rollup` in watch mode.
 
-2. Run `pnpm link --global` in `packages/vite`. This links `vite` globally so that you can:
+1. Run `pnpm link --global` in `packages/vite`. This links `vite` globally so that you can:
```

Comment:
I did not know this trick of using 1. for all numbers is a markdown, awesome! 🎉

Reply:
Yeah, but lets stick to normal human readable number sequence. So the file is also reabable when it's not rendered

Reply:
done!